etl/steps/data/garden/regions/2023-01-01/regions.yml

```diff
@@ -1847,3 +1847,221 @@
   end_year: 1902
   successors:
     - "ZAF"
+
+# WHO regios
+- code: WHO_AMR
```

Comment:
I'd add `defined_by: who` to all entries. That way, we have a field to distinguish between our and other institutions' definitions.